src/backup_restore.cpp

```diff
@@ -128,10 +128,165 @@ void BackupRestore::backupAndRestoreIpzVpd(types::IPZVpdMap& io_srcVpdMap,
                                            const std::string& i_srcPath,
                                            const std::string& i_dstPath)
 {
-    (void)io_srcVpdMap;
-    (void)io_dstVpdMap;
-    (void)i_srcPath;
-    (void)i_dstPath;
+    if (!m_backupRestoreCfgJsonObj.contains("backupMap") ||
+        !m_backupRestoreCfgJsonObj["backupMap"].is_array())
+    {
+        logging::logMessage(
+            "Backup map not found in the backup and restore config JSON.");
+        return;
+    }
+
+    bool l_isEmptySrcVpdReceived = io_srcVpdMap.empty();
+    bool l_isEmptyDstVpdReceived = io_dstVpdMap.empty();
+
+    for (const auto& l_aRecordKwInfo : m_backupRestoreCfgJsonObj["backupMap"])
+    {
+        const std::string& l_srcRecordName =
+            l_aRecordKwInfo.value("sourceRecord", "");
+        const std::string& l_srcKeyword = l_aRecordKwInfo.value("sourceKeyword",
```

Comment:
> l_srcKeyword

l_srcKeywordName, here and everywhere else.

Reply:
Updated.